### ISSUE TITLE
Giving to the User the option to decide which subset should be used as Dev Set --rather than using cross-validation.

### ISSUE THREAD

**Issue by AlexanderKonstantinidis** [pycaret/pycaret#156]
PyCaret implements a very promising idea.  Unfortunately, the PyCaret Module by default uses cross-validation for Hyperparameter optimization and model evaluation without offering any alternative.  However, there are cases where this is not advisable.  Specifically whenever one feels that the entire training set is not equally representative of the out of sample set.  In such cases, one would benefit by using as Dev Set a subset of the Training Set that most resembles the Out-of-Sample Set. 
A special case of this general category is whenever the examples represent observations with a temporal dimension inherent.  In such a case, it would be advisable to use as Dev Set a subset of examples that is temporally subsequent to the Train Set.
The issue I raise is then:  Provide an option to the user to override the cross-validation default and use instead a Dev Set (Validation Set) of the user's choice.

**Comment by pycaret** [comment]
@AlexanderKonstantinidis This is a great idea. Marking this as enhancement for future. Please feel free to take a stab on it and submit and PR on dev-1.0.1 branch.

**Comment by pycaret** [comment]
@AlexanderKonstantinidis 

This issue has been fixed in the `pycaret-nightly` release. Please install pycaret-nightly using `pip install pycaret-nightly`. Feel free to open a new issue, in case the problem persists. This fix will be published to PyPi as `pycaret==2.2.0` on October 28, 2020.
